tests/system/python/packages/test_available_and_install_api.py

```diff
@@ -20,8 +20,9 @@
 __version__ = "${VERSION}"
 
 available_pkg = []
-# plugins installed by default 4
 counter = 4
+"""  By default 4 plugins are installed i.e. all north
+"""
```

Comment:
Better to add `TODO: FOGL-3168`, once resolved we need to update the test as by default only 2 north plugins

Reply:
I would rather like to see it failing, when the real behavior changes.